pkg/setup/flags.go

```diff
@@ -39,6 +40,12 @@ func (f *Flags) Register(flags *pflag.FlagSet) {
 		false,
 		"Do not check CA certficate downloaded from cluster (insecure). Applies to Enterprise DC/OS only.",
 	)
+	flags.BoolVar(
+		&f.noPlugin,
+		"no-plugin",
+		false,
+		"Do not auto-install dcos-core-cli and dcos-enterprise-cli plugins.",
```

Comment:
A little more descriptive: `Do not install the core and enterprise plugins locally after setup`.

Reply:
I don't think it's better, the auto-installation would now be part of the setup, not after, it just happens to be the last step.